packages/react-components/src/components/VerticalGallery.tsx

```diff
@@ -118,9 +118,12 @@ export const VerticalGallery = (props: VerticalGalleryProps): JSX.Element => {
   const firstIndexOfCurrentPage = (page - 1) * childrenPerPage;
   const clippedPage = firstIndexOfCurrentPage < numberOfChildren - 1 ? page : lastPage;
   const childrenOnCurrentPage = useMemo(() => {
-    return indexesArray[clippedPage - 1].map((index) => {
-      return React.Children.toArray(children)[index];
-    });
+    if (indexesArray[0] !== undefined) {
```

Comment:
Should we check length of index array rather than first element being undefined

Reply:
Because this is a doubly nested array it will always have a truthy reference so we need to check to see if the first index is defined to do that undefined check.